### PR TITLE
[FIX](load)fix load with split-by-string

### PR DESCRIPTION
**Pull request by amorynan**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/load/Load.java [hunk 1]
```diff
@@ -553,9 +553,9 @@ private static void analyzeAllExprs(Table tbl, Analyzer analyzer, Map<String, Ex
 
             // Array type do not support cast now
```
Comment (eldenmoon) [inline fe/fe-core/src/main/java/org/apache/doris/load/Load.java:554]: comment should be modified

Comment (amorynan) [reply fe/fe-core/src/main/java/org/apache/doris/load/Load.java]: done!

**Comment by amorynan** [comment]
run buildall

**Review by eldenmoon — APPROVED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**